azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/ResourceGroup.java

```diff
@@ -1,7 +1,7 @@
 package com.microsoft.azure.management.resources;
 
 import com.microsoft.azure.management.resources.fluentcore.arm.Region;
-import com.microsoft.azure.management.resources.fluentcore.arm.models.Taggable;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.Resource;
 import com.microsoft.azure.management.resources.fluentcore.model.*;
```

Comment:
note the import ".*" stuff is not part of my checkin. it was already there. I agree it should be explicit.